### ISSUE TITLE
Inconsistency between RotateTransform and TransformOperations

### ISSUE THREAD

**Issue by lucdem** [AvaloniaUI/Avalonia#14615]
## Describe the bug
The RotateTransform class takes an angle in degrees as it's parameter, while the TransformOperations.Builder.AppendRotate method receives an angle in radians, which is confusing and inconsistent.

The RotateTransform constructor has documentation specifying that the parameter in in degrees, but to find out whether the TransformOperations builder uses degrees or radians one needs to dig into the source code.

## To Reproduce

Create and app with an image inside a LayoutTransformControl, apply one of the 2 versions in code-behind:

```
// v1
MyLayoutTransformControl.LayoutTransform = new RotateTransform(90);

// equivalent to v1
var builder = TransformOperations.CreateBuilder(1);
builder.AppendMatrix(new RotateTransform(90));

// v2
var builder = TransformOperations.CreateBuilder(1);
builder.AppendRotate(90);
MyLayoutTransformControl.LayoutTransform = builder.Build();
```

## Expected behavior
Most would assume v1 and v2 to both rotate the image by 90 degrees (or both by 90 rad), but only v1 does so. Ideally both would use radians, or both use degrees, or maybe both should receive an Angle struct with factory methods "FromDegrees" and "FromRadians", however those would be breaking changes.

If the inconsistency is considered not enough to justify a breaking change, then at minimum the TransformOperations.Builder should have clearer documentation on the AppendRotate method. 

## Environment 

- OS: Windows
- Avalonia-Version: 11.0.7

**Comment by maxkatz6** [comment]
Yes, documentation should be clear there. Right now, is completely missing on TransformOperations.Builder.
Possibly having second method - `builder.AppendRotateInDegrees` would also help making these methods more distinct.
